sdk/translation/Azure.AI.Translation.Document/src/DocumentTranslationOperation.cs

```diff
@@ -545,11 +545,11 @@ private void ValidateOperationStatus()
                 throw _requestFailedException;
         }
 
-        private static IDictionary<string, string> CreateAdditionalInformation(DocumentTranslationError error)
+        private static IDictionary<string, string> CreateAdditionalInformation(ResponseError error)
         {
-            if (string.IsNullOrEmpty(error.Target))
+            if (string.IsNullOrEmpty(error.ToString()))
                 return null;
-            return new Dictionary<string, string> { { "Target", error.Target } };
+            return new Dictionary<string, string>(1) { { "AdditionInformation", error.ToString() } };
```

Comment:
Does AdditionalInformation == inner error + details?

Reply:
```suggestion
            return new Dictionary<string, string>(1) { { "AdditionalInformation", error.ToString() } };
```

Just highlighting the area where Krista called out the typo;  it took me a couple of scans to see it.

Reply:
I forgot to reply. Sorry!!

> Does AdditionalInformation == inner error + details?

Yes. It adds everything it can find. Even the `Target` property